docs/system-data-in-properties-bag.md

```diff
@@ -0,0 +1,48 @@
+# SystemDataInPropertiesBag
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-SystemData-V1-01
+- RPC-SystemData-V1-02
+
+## Description
+
+Validates that system data is not defined in the properties bag, but rather as a top-level property.
+
+## How to fix the violation
+
+Ensure that any system data definitions are as top-level properties, not in the properties bag.
+
+### Valid/Good Example
+
+```json
+"Resource": {
+  "properties": {
+    "systemData": {
+      "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+    }
+  }
+}
+```
+
+### Invalid/Bad Example
+
+```json
+"Resource": {
+  "properties": {
+    "properties": {
```

Comment:
Is it common to have nested `properties/properties` ? If not, maybe let's use names that are more realistic?

Reply:
OK I found below

`given: ["$.definitions.*.properties.properties.systemData", "$.definitions.*.properties.properties.SystemData"],`

so I guess `properties.properties` is the standard.

Reply:
Yes, the first level of `properties` is part of the OpenAPI specification syntax and the second level is describing the Resource to have a field called `properties`.